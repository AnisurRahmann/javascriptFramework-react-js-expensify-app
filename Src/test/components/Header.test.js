
import {shallow} from 'enzyme';
import React from 'react';
import Header from '../../components/Header';
test('test header components',()=>{
    const wrapper = shallow(<Header/>);
    expect(wrapper).toMatchSnapshot();
    
    //expect(wrapper.find('h1').text()).toBe('Expensify');
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header/>);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})