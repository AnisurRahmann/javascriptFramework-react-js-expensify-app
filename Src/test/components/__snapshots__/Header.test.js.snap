// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header components 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    dasheBoadrd
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/add"
  >
    Creat Expenses
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
